instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/servlet/ServletContextPath.java

```diff
@@ -22,18 +23,41 @@
 
   // Keeps track of the servlet context path that needs to be prepended to the route when updating
   // the span name
-  public static final ContextKey<String> CONTEXT_KEY =
+  private static final ContextKey<ServletContextPath> CONTEXT_KEY =
```

Comment:
do we need the `ServletContextPath` wrapper object?

Reply:
Using wrapper helps to distinguish context path not yet set from context path empty. I guess I could use `""` to mark empty context path.